config-model/src/test/java/com/yahoo/config/model/provision/ModelProvisioningTest.java

```diff
@@ -1408,10 +1469,14 @@ public void testMultitenantButNotHostedSharedContentNode() {
     }
 
     private VespaModel createNonProvisionedMultitenantModel(String services) {
-        VespaModelCreatorWithMockPkg modelCreatorWithMockPkg = new VespaModelCreatorWithMockPkg(null, services, ApplicationPackageUtils.generateSearchDefinition("type1"));
+        return createNonProvisionedModel(true, null, services);
+    }
+
+    private VespaModel createNonProvisionedModel(boolean multitenant, String hosts, String services) {
```

Comment:
Calling the param `isMultitenant` makes it easier to read and obvious that it's a boolean.

Reply:
I like that for fields not really for parameters (because it is an imperative ... more like makeItMultitenant than isMultitenant).